crates/vfio-bindings/src/bindings_v5_0_0/vfio.rs

```diff
@@ -1,5 +1,7 @@
 /* automatically generated by rust-bindgen */
 
+#![cfg_attr(test, allow(deref_nullptr))]
```

Comment:
Did you add this by hand? This will be overwritten once a new binding file is generated.

Also, there is already `#![allow(clippy::all)]` in `mod.rs`. Is that not enough?

Reply:
Thanks for pointing this out! Yes, I did thus by hand similar to https://github.com/rust-vmm/vmm-reference/blob/main/src/vm-vcpu-ref/src/x86_64/mpspec.rs#L15

I guess I was hoping for this thing to be fixed in `rust-bindgen` upstream before it's going to be used again.

`#![allow(clippy::all)]` is probably only for the `mod.rs` file only and not for the submodules, although I'm not 100% sure. Let me check.

Reply:
I think the warnings are not generated by clippy, but by rustc. You can add the change to `mod.rs`. That prevents it from getting overwritten.

Fundamentally I think those complains are valid. I'm wondering if a newer version of bindgen can generate code that contains no UB.

Reply:
The issue in bindgen is still open https://github.com/rust-lang/rust-bindgen/issues/1651

Reply:
Please also put the link to the PR into a comment before the cfg. We will know when the workaround can be dropped. Thanks.